src/Microsoft.DotNet.Build.Tasks/CreateAzureFileShare.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.File;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public sealed class CreateAzureFileShare : Task
+    {
+        [Required]
+        public string AccountName { get; set; }
+
+        [Required]
+        public string AccountKey { get; set; }
+
+        [Output]
```

Comment:
It looks a little strange to have an output parameter mixed in with the input ones, though I understand that it's because that's alphabetically correct...